pkg/loop/config.go

```diff
@@ -114,3 +123,16 @@ func getTracingSamplingRatio() float64 {
 	}
 	return samplingRatio
 }
+
+// getTLSCertPath parses and validates the CL_TRACING_TLS_CERT_PATH environment variable.
+func getTLSCertPath() (string, error) {
+	// empty string is valid; signals to use insecure credentials
```

Comment:
i'm confused by this comment and what happens if this is empty.

would LookupEnv be a better fit https://pkg.go.dev/os#LookupEnv  so we can distinguish unset value?

Reply:
Valid TOML:

```
[Tracing]
Enabled = true
CollectorTarget = 'localhost:4317'
TLSCertPath = '/path/to/cert.pem'
```

Valid TOML:

```
[Tracing]
Enabled = true
CollectorTarget = 'localhost:4317'
# Note that TLSCertPath is not specified
```

Invalid TOML:
```
[Tracing]
Enabled = true
CollectorTarget = 'localhost:4317'
TLSCertPath = ''
```
The 2nd option is how I intend users specify insecure connections to the otel-collector are allowed. From a security perspective, this is fine when the nodes and the otel-collector live on the same network.

LookupEnv would be able to tell when the plugin is booted whether or not a value is set, but this doesn't matter because a value of empty string will be caught by the host before it is sent over to the plugin via env var.